shared/src/main/scala/minitest/api/Result.scala

```diff
@@ -69,7 +69,7 @@ object Result {
       val description = {
         val name = source.getClass.getName
         val className = name.substring(name.lastIndexOf(".") + 1)
-        Option(source.getMessage).filterNot(_.isEmpty)
+        Option(source.getMessage).filterNot(_.nn.isEmpty)
```

Comment:
Requires fix to compiler: Option(something nullable) is Option[something nullable]; should be Option[non-null]